tests/test_toctree.py

```diff
@@ -20,8 +20,9 @@ def test_relations(app, status, warning):
     assert app.builder.relations['bar/index'] == ['index', 'foo', 'bar/bar_1']
     assert app.builder.relations['bar/bar_1'] == ['bar/index', 'bar/index', 'bar/bar_2']
     assert app.builder.relations['bar/bar_2'] == ['bar/index', 'bar/bar_1', 'bar/bar_3']
-    assert app.builder.relations['bar/bar_3'] == ['bar/index', 'bar/bar_2', 'bar/bar_4/index']
-    assert app.builder.relations['bar/bar_4/index'] == ['bar/index', 'bar/bar_3', 'baz']
+    assert app.builder.relations['bar/bar_3'] == ['bar/index', 'bar/bar_2', 'bar/bar_10']
+    assert app.builder.relations['bar/bar_10'] == ['bar/index', 'bar/bar_3', 'bar/bar_4/index']
+    assert app.builder.relations['bar/bar_4/index'] == ['bar/index', 'bar/bar_10', 'baz']
```

Comment:
We need to discuss this is intended behavior for Sphinx. It might be needed to develop natsort for Sphinx (for toctree).

Reply:
Agreed. I'll work to make a custom sort module for Sphinx. BTW, do you have any opinion to introduce natural sort (without new option)?

Reply:
> option

I don't think it's necessary.
I believe that when document authors use numbers for each files, they will want to naturally sort it.